flexget/plugins/sites/horriblesubs.py

```diff
@@ -22,8 +22,13 @@ class HorribleSubs(object):
     @staticmethod
     def horrible_entries(requests, page_url):
         entries = []
-        soup = get_soup(requests.get(page_url).content)
-        
+
+        try:
+            soup = get_soup(requests.get(page_url).content)
+            log.debug('requesting: %s', page_url)
```

Comment:
https://github.com/Flexget/Flexget/blob/next/flexget/utils/requests.py#L240 <-- this already logs the request url